### ISSUE TITLE
Add Penalization

### ISSUE THREAD

**Issue by ttiagojm** [ttiagojm/DBN-TF2#3]
### Problem
In the article they recommend us to use a regularization, specifically, a L2 regularization to penalize errors and prevent overfitting.

### Solution
Implement the expression of a L2 regularization (or use a built-in function from tensorflow).
Using Dropout instead of L2 is a possibility even if they don't mention that in the article.
